test/PausableAdminUpgradeable.js

```diff
@@ -0,0 +1,205 @@
+const { config, ethers, network } = require("hardhat");
+const { expect } = require("chai");
+
+describe("MockPausableAdminUpgradeable", function () {
```

Comment:
Add a failing test trying to initialize again. (that way you would spot if you'd have forgotten the onlyIntialize modfier)